<launch>
    <arg name="bag_path" default="~/Desktop" />
    <arg name="bag_name_prefix" default="exploration" />
    <arg name="robot_id" default="0" />
    <arg name="robot_num" default="1" />
    <node pkg="rosbag" type="record" name="rosbag_record_tare_planner" args="record -o $(arg bag_path)/robot$(arg robot_id)_$(arg robot_num)_$(arg bag_name_prefix) -e '/sensor_coverage_planner/(.*)|(.*)exploration_info(.*)|(.*)position' /tf /registered_scan /state_estimation /state_estimation_at_scan /velodyne_cloud_registered /integrated_to_init /aft_mapped_to_init /way_point /runtime /rosout" >
      <param name="/use_sim_time" value="true"/>
    </node>
</launch>